packages/wifi_iot/lib/wifi_iot.dart

```diff
@@ -5,13 +5,7 @@ import 'dart:io';
 
 import 'package:flutter/services.dart';
 
-enum WIFI_AP_STATE {
```

Comment:
Can we do without the reformatting please?

Reply:
@percymudzinga merging https://github.com/percymudzinga/WiFiFlutter/pull/1 should fix this.

Reply:
Looks like I can do it myself. Resolving this. done with https://github.com/florian-guillemard/WiFiFlutter/pull/215/commits/703dad27d09ab2f25be2e353836cf4f66b4975e6